Makefile

```diff
@@ -49,10 +49,8 @@ ifeq ($(STATIC),1)
 TAGS += netgo
 # The installsuffix makes sure we actually get the netgo build, see
 # https://github.com/golang/go/issues/9369#issuecomment-69864440
-GOFLAGS  += -installsuffix netgo
-LDFLAGS += -extldflags "-lm -lstdc++ -static"
-# rebuild everything.
-GOFLAGS += -a
+GOFLAGS += -installsuffix netgo
+LDFLAGS += -extldflags "-static"
```

Comment:
Yes, I've tried this in a docker container and `ldd` reports the binary is statically linked.